experiments/compositions/composition/config/expanders/expander_versions.yaml

```diff
@@ -16,12 +16,27 @@ apiVersion: composition.google.com/v1alpha1
 kind: ExpanderVersion
 metadata:
   name: jinja2
-  namespace: composition-system
+  namespace: system
```

Comment:
did we change the namespace from composition-system to system as well?

Reply:
the config/expanders can have `system`.
composition is added when we run kustomize

Reply:
It will add to the namespace of other CRs too? I thought it just add to the name, like jinja2 will become composition-jinja2. But if it does add to the namespace, you can resolve this comment.

Reply:
https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/experiments/compositions/composition/config/release/kustomization.yaml#L16

All objects end up in `composition-system`